pom.xml

```diff
@@ -20,6 +20,12 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.moditect</groupId>
+    <artifactId>moditect-org-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
   <groupId>dev.morling.jfrunit</groupId>
```

Comment:
Let's change that, too. `org.moditect.jfrunit`, or `org.moditect`?

Reply:
I'd suggest `org.moditect.jfrunit` to keep it separate from the `moditect` project. Who knows, perharps JfrUnit will have additional artifacts in the future thus having its own namespace is a good thin.